### PR TITLE
Wrote test for word count.

### PR DESCRIPTION
**Pull request by Calvyn82**
Here is an example of what I had in mind.

### CODE REVIEW

**Comment by kotp** [comment]
I think it needs the code as to pass the tests as well.  This will give us an idea if it shifts the target problem enough to create a new exercism problem. 

re #176  this does not address the parenthesis part of your issue title.  Perhaps the title is wrong?

**Comment by Calvyn82** [comment]
It is. I don't even have the excuse of English not being my first language. So sorry.

**Comment by Calvyn82** [comment]
Did you mean that I needed to update the example code to make it pass the test as well?

If so, I updated it. The example now passes all of the tests.

File: word-count/word_count_test.rb [hunk 1]
```diff
@@ -82,4 +82,14 @@ def test_with_apostrophes
...
   end
+
+  def test_with_quotations
+    skip
+    phrase = Phrase.new("Joe can't tell between 'large' and large.")
+    counts = {
+      'joe' => 1, "can't" => 1, 'tell' => 1,
+      'between' => 1, 'large' => 2, 'and' => 1
+    }
+    assert_equel counts, phrase.word_count
```
Comment (kotp) [inline word-count/word_count_test.rb]: assert_equal is misspelled so Travis is failing.

**Comment by Calvyn82** [comment]
Thought I had that fixed. Must have reverted it when I put the skips back in. Should work now.

**Comment by kotp** [comment]
Looks good to me, @kytrinyx you think the wording should change in the exercise description?  It is now looking for only 'proper' words exclusive of punctuation now.

I think it is close enough to the original intent of the exercise.  We aren't trying to do sentence count, etc, nor looking at hyphenated words.  This does address a pretty simple idea of what we would look for in a word count program, I think.

**Comment by kotp** [comment]
Version information should be added in though.

**Comment by Calvyn82** [comment]
I'm getting the x-common stuff setup and looked at right now for the versioning.

**Comment by kytrinyx** [comment]
> I think it is close enough to the original intent of the exercise.

Yeah, agreed.


About the versioning--why did you remove it? If you add the constant to the example solution, it should pass the tests.

**Comment by Calvyn82** [comment]
I'm open to suggestions about what I'm doing wrong, here.

**Comment by kotp** [comment]
I am not sure that the `VERSION` constant was there yet, @kytrinyx .  So not sure that it was removed.

@Calvyn82 Can you look at another exercise set to see an example of how the VERSION constant works?

**Comment by Calvyn82** [comment]
So, In line 7 of hamming/hamming_test.rb it has a commented out merge statement.
# 11a7e6f Merge pull request #92 from kotp/116_Verbal_Word_Problem

 Is that something Travis needs for word-count/word_count_test.rb? If so, what pull request should I match it up with?

**Comment by kotp** [comment]
The current (master) branch is what you should compare with.  That comment is just a comment.  It is not read by anything to produce anything.

**Comment by kytrinyx** [comment]
The comment is used by the generator to show what version of the shared data it's using. You don't need to pay attention to that.

**Comment by kytrinyx** [comment]
The failing build is unrelated to this--it's updates to the rubocop gem (and we haven't pinned it). Once this is squashed, it's safe to merge. I'll go fix the rubocop stuff in a separate branch.

**Pull request merged**